tools/encodings/GF 2016 Glyph Sets/FilterLists/Google Latin Expert.txt

```diff
@@ -372,6 +372,15 @@ breveinvertedcomb
 dblgravecomb
 horncomb
 dotbelowcomb
+strokeshortcomb
+brevecomb_acutecomb
+brevecomb_gravecomb
+brevecomb_hookabovecomb
+brevecomb_tildecomb
+circumflexcomb_acutecomb
+circumflexcomb_gravecomb
+circumflexcomb_hookabovecomb
+circumflexcomb_tildecomb
```

Comment:
I'm not sure if all these are needed. These can be accessed through mark2mark attachment. Unless i'm missing something?